externs/fbjs.ext.js

```diff
@@ -0,0 +1,30 @@
+/** @suppress {duplicate} */
```

Comment:
Why is this file required?

Reply:
This is only needed for the node bundles because the contents of fbjs are not included in those bundles, so properties like `emptyFunction.thatReturns` get renamed when they shouldn't be. I also remember hitting a problem with `canUseDOM` not being set to true when it should be, but I can't repro that right now. I was being defensive when I created this file by including everything from fbjs that I saw being used, but we can cut it down to a smaller set of things if that would be preferable.

Reply:
Forgot to mention — this file is not pulled in for UMD bundles, because fbjs is not part of `externals` for those, which are checked in `getClosureExterns` to determine which externs files to look for.

Reply:
I got it. Thank you!